app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/model/MovieResponse.kt

```diff
@@ -0,0 +1,3 @@
+package com.example.marcinwisniewski.intivetrainingapp.model
+
+data class MovieResponse(var results: List<Movie>?)
```

Comment:
why var? I think the class should be immutable, use val instead.

Reply:
Done in both ways